package.json

```diff
@@ -129,5 +130,6 @@
   "resolutions": {
     "jackspeak": "2.1.1",
     "@radix-ui/react-id": "1.1.0"
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
This seems a little too precise, what if I have 

```
 ~  yarn --version                                                                                                                                 479ms  Wed 10 Jul 13:33:52 2024
1.22.22
```

Also seems unrelated to this PR

Reply:
grr, that's my local yarn which added this itself for some reason :(
I think it's corepack related or something?

Reply:
Removed